salt/metalk8s/defaults.yaml

```diff
@@ -6,3 +6,13 @@ repo:
     version: latest
   kubernetes:
     version: latest
+kubelet:
+  config_file: "/var/lib/kubelet/config.yaml"
+  config:
+    apiVersion: kubelet.config.k8s.io/v1beta1
```

Comment:
Don't know how we can infer this one

Reply:
What do you mean with 'infer'? From what?

Reply:
I don't know, it's depend of the version of kubelet but maybe we stuck to a specific config apiVersion for a given version of `metalk8s`

Reply:
Partially discussed offline :) It goes with a version of `kubelet`, so it also goes with a specific version of MetalK8s.